features/data/events/rdf/event-with-periodic-calendar-and-opening-hours.ttl

```diff
@@ -33,21 +33,19 @@
     cidoc:P161_has_spatial_projection <http://data.uitdatabank.local:80/places/%{uuid_place}> ;
     cidoc:P160_has_temporal_projection [
       a m8g:PeriodOfTime ;
-      m8g:startTime ""^^xsd:dateTime ;
-      m8g:endTime ""^^xsd:dateTime
+      m8g:startTime "2021-05-17T08:00:00+00:00"^^xsd:dateTime ;
+      m8g:endTime "2021-05-18T22:00:00+00:00"^^xsd:dateTime
     ]
   ] ;
-  cpa:beschikbaarheid [
-    a cp:Beschikbaarheid ;
-    schema:hoursAvailable [
-      a schema:OpeningHoursSpecification ;
-      schema:opens "10:00" ;
-      schema:closes "16:00" ;
-      schema:dayOfWeek schema:Monday, schema:Tuesday
-    ], [
-      a schema:OpeningHoursSpecification ;
-      schema:opens "09:00" ;
-      schema:closes "17:00" ;
-      schema:dayOfWeek schema:Saturday, schema:Sunday
-    ]
-  ] .
\ No newline at end of file
+  cpa:beschikbaarheid <http://data.uitdatabank.local:80/events/%{eventId}#Beschikbaarheid-2450812e> .
+
+<http://data.uitdatabank.local:80/events/%{eventId}#Beschikbaarheid-2450812e> schema:hoursAvailable <http://data.uitdatabank.local:80/events/%{eventId}#OpeningHoursSpecification-19cb130c>, <http://data.uitdatabank.local:80/events/%{eventId}#OpeningHoursSpecification-74f53a6d> .
```

Comment:
One of the following examples have these URIs on new lines, can that also be done here because that greatly improves readability

Reply:
I have no idea how to this.
It just use the outputs from the library, I have no control over this.
This would break the test afaik.

Reply:
I just tried entering some enters but like expected that would break the tests.